src/ceph-volume/plugin/cache/ceph_volume_cache/main.py

```diff
@@ -0,0 +1,207 @@
+import argparse
+import sys
+from ceph_volume.api import lvm as api
+from ceph_volume.util import disk
+
+"""
+The user is responsible for splitting the disk into data and metadata partitions.
+
+If we are to partition a disk to be used as a cache layer, no partition can be
+smaller than 2GB because ceph-volume creates vgs with PE = 1GB.
+
+"""
+
+
+# Not very elegant, probably needs to be replaced or at least moved to util
+def strToSize(s):
+    n = float(s[:-1])
```

Comment:
If the function accepts bytes, i.e. without prefixes, then the Size will be incorrect.